Godeps/Godeps.json

```diff
@@ -77,18 +77,13 @@
 		},
 		{
 			"ImportPath": "github.com/docker/docker/pkg/tarsum",
-			"Comment": "v1.4.1-775-g70fbd45",
-			"Rev": "70fbd45a5c88f6f39a07b04f81a07721bf5f3eed"
+			"Comment": "v1.7.1",
+			"Rev": "786b29d4db80a6175e72b47a794ee044918ba734"
```

Comment:
`github.com/docker/distribution/digest`

We can probably deprecate it in that package since we never shipped a distribution release with support for it, and older docker clients don't use that package. Would file an issue in distribution to change it.